package.json

```diff
@@ -5,60 +5,34 @@
     "url",
     "uri"
   ],
-  "version": "1.0.0",
+  "version": "2.0.0",
   "bugs": {
     "url": "https://github.com/cainus/urlgrey/issues"
   },
   "scripts": {
-    "test": "make test-coveralls"
-  },
-  "browser": {
-    "url": "./browser/url.js",
-    "querystring": "./browser/querystring.js"
+    "test": "jest"
   },
   "maintainers": [
     "Gregg Caines <gregg@caines.ca> (http://caines.ca)"
   ],
-  "dependencies": {
-    "fast-url-parser": "^1.1.3"
```

Comment:
I'd recommend benchmarking this removal. There's a huge difference between it and the Node.JS url library. Using fast-url-parser significantly reduced how much event-loop-blocking we were seeing within API when handling 1,000-10,000s of urls.